Remove str float z for section

mahaling opened the pull request:
Removing str(float(z)) for section ID... The section IDs will default to what was set while ingestion of tilespecs, unless "remap_section_ids" option is set to True.

RussTorres commented:
As previously mentioned, the commits up to now are even with the develop branch, so it would probably be nice to roll those back/squash them into a commit that makes a code change.  The changes that I can think of for this PR are:

https://github.com/AllenInstitute/render-modules/blob/e2dc6ebb39cca94a784f635f2748dd747766eb09/rendermodules/dataimport/make_montage_scapes_stack.py#L132

https://github.com/AllenInstitute/render-modules/blob/e2dc6ebb39cca94a784f635f2748dd747766eb09/integration_tests/test_rough_align.py#L1150

https://github.com/AllenInstitute/render-modules/blob/e2dc6ebb39cca94a784f635f2748dd747766eb09/integration_tests/test_rough_align.py#L1161

mahaling commented:
> As previously mentioned, the commits up to now are even with the develop branch, so it would probably be nice to roll those back/squash them into a commit that makes a code change. The changes that I can think of for this PR are:
> 
> [render-modules/rendermodules/dataimport/make_montage_scapes_stack.py](https://github.com/AllenInstitute/render-modules/blob/e2dc6ebb39cca94a784f635f2748dd747766eb09/rendermodules/dataimport/make_montage_scapes_stack.py#L132)
> 
> Line 132 in [e2dc6eb](/AllenInstitute/render-modules/commit/e2dc6ebb39cca94a784f635f2748dd747766eb09)
> 
>  t.layout.sectionId = "%s.0" % str(int(newz)) 
> [render-modules/integration_tests/test_rough_align.py](https://github.com/AllenInstitute/render-modules/blob/e2dc6ebb39cca94a784f635f2748dd747766eb09/integration_tests/test_rough_align.py#L1150)
> 
> Line 1150 in [e2dc6eb](/AllenInstitute/render-modules/commit/e2dc6ebb39cca94a784f635f2748dd747766eb09)
> 
>  assert(int(z) == int(float(sectionid))) 
> [render-modules/integration_tests/test_rough_align.py](https://github.com/AllenInstitute/render-modules/blob/e2dc6ebb39cca94a784f635f2748dd747766eb09/integration_tests/test_rough_align.py#L1161)
> 
> Line 1161 in [e2dc6eb](/AllenInstitute/render-modules/commit/e2dc6ebb39cca94a784f635f2748dd747766eb09)
> 
>  assert(int(z) == int(float(sectionid)))

The first change is probably not required as that stack and point matches never get used beyond rough alignment. Still I changed it. Just so we don't leave any trace of int(float(sectionid))

changed the other areas as well.

RussTorres commented on integration_tests/test_rough_align.py:
```diff
@@ -84,7 +84,7 @@ def montage_stack(render, resolvedtiles_from_json):
 
     yield test_montage_stack
-    renderapi.stack.delete_stack(test_montage_stack, render=render)
+    #renderapi.stack.delete_stack(test_montage_stack, render=render)
```
These stacks should still be deleted during cleanup.

mahaling replied:
done

djkapner commented:
Just searching the repo for "`str(float(" I also see:
rendermodules/pointmatch_filter/filter_point_matches.py
and
rendermodules/mesh_lens_correction/do_mesh_lens_correction.py

did you look at these and determine they were ok?

RussTorres commented on integration_tests/test_rough_align.py:
```diff
@@ -297,7 +297,7 @@ def montage_z_mapped_stack(render, montage_stack, downsample_sections_dir):
 
     yield output_stack
-    renderapi.stack.delete_stack(output_stack, render=render)
+    #renderapi.stack.delete_stack(output_stack, render=render)
```
```suggestion
    renderapi.stack.delete_stack(output_stack, render=render)
```
